src/hotspot/os/windows/sharedRuntimeRem.cpp

```diff
@@ -38,7 +39,7 @@
 
 static const double one = 1.0, Zero[] = { 0.0, -0.0, };
 
-double SharedRuntime::fmod_winx64(double x, double y)
+double SharedRuntime::fmod_win64(double x, double y)
```

Comment:
The name was suggested by Andrew, I'll change it to fmod\_winarm64 if you prefer

Reply:
I thought at the time the name was because it was for both x64 and aarch64 - hence any win64 platform. But if it is only for Aarch64 then it makes sense to me that the function name conveys that.

Reply:
Alright, I'll change it